Tools/CISupport/ews-build/steps.py

```diff
@@ -1642,6 +1665,37 @@ def doStepIf(self, step):
         return CURRENT_HOSTNAME == EWS_BUILD_HOSTNAME
 
 
+class CommentOnPullRequest(buildstep.BuildStep, GitHubMixin):
```

Comment:
Why not just combine CommentOnBug and this one together?

Reply:
I can do that. What would such a class be called? `CommentOn`?

Reply:
not sure, maybe `CommentOnChange` or `LeaveComment`